### PR TITLE
Translate getting started

### PR DESCRIPTION
**Pull request by jin-s13**
1. Fix some errors in the docs.
2. Translate it into Chinese.

### CODE REVIEW

File: docs_zh-CN/getting_started.md [hunk 1]
```diff
@@ -1,3 +1,249 @@
...
+MMPose 提供了一些测试脚本用于测试数据集上的指标（如 COCO, MPII 等），
+并提供了一些高级 API，以便更好地兼容其他 OpenMMLab 项目。
+
+### 测试某个数据集
+
+- [x] 支持单 GPU
+- [x] 支持单节点，多 GPU
+- [x] 支持多节点
+
+用户可使用以下命令进行数据集测试
```
Comment (innerlee) [inline docs_zh-CN/getting_started.md:46]: 用户可使用以下命令测试数据集

Be aware of the pattern `进行xxx`. Just `xxx` it.

File: docs/getting_started.md [hunk 2]
```diff
@@ -209,7 +220,7 @@ CUDA_VISIBLE_DEVICES=0,1,2,3 PORT=29500 ./tools/dist_train.sh ${CONFIG_FILE} 4
 CUDA_VISIBLE_DEVICES=4,5,6,7 PORT=29501 ./tools/dist_train.sh ${CONFIG_FILE} 4
 ```
 
-If you use launch training jobs with slurm, you need to modify the config files (usually the 6th line from the bottom in config files) to set different communication ports.
+If you use launch training jobs with slurm, you need to modify the config files (usually the 4th line in config files) to set different communication ports.
```
Comment (innerlee) [inline docs/getting_started.md:223]: Unrelated to this translation. We should randomize the port number

File: docs_zh-CN/getting_started.md [hunk 3]
```diff
@@ -1,3 +1,249 @@
...
+```
+
+此处的 `CHECKPOINT_FILE` 可以是本地的模型权重文件的路径，也可以是模型的下载链接。
+
+可选参数:
+
+- `RESULT_FILE`：输出结果文件名。如果没有被指定，则不会保存测试结果。
+- `--fuse-conv-bn`: 是否融合BN和Conv层。该操作会略微提升模型推理速度。
+- `EVAL_METRICS`：测试指标。其可选值与对应数据集相关，如 `top_k_accuracy`，`mean_class_accuracy` 适用于所有动作识别数据集，`mmit_mean_average_precision` 适用于 Multi-Moments in Time 数据集，`mean_average_precision` 适用于 Multi-Moments in Time 和单类 HVU 数据集，`AR@AN` 适用于 ActivityNet 数据集等。
+- `--gpu-collect`：如果被指定，动作识别结果将会通过 GPU 通信进行收集。否则，它将被存储到不同 GPU 上的 `TMPDIR` 文件夹中，并在 rank 0 的进程中被收集。
```
Comment (innerlee) [inline docs_zh-CN/getting_started.md:67]: 动作识别

File: docs_zh-CN/getting_started.md [hunk 4]
```diff
@@ -1,3 +1,249 @@
...
+    [--launcher ${JOB_LAUNCHER}] [--local_rank ${LOCAL_RANK}]
+```
+
+此处的 `CHECKPOINT_FILE` 可以是本地的模型权重文件的路径，也可以是模型的下载链接。
+
+可选参数:
+
+- `RESULT_FILE`：输出结果文件名。如果没有被指定，则不会保存测试结果。
+- `--fuse-conv-bn`: 是否融合BN和Conv层。该操作会略微提升模型推理速度。
+- `EVAL_METRICS`：测试指标。其可选值与对应数据集相关，如 `top_k_accuracy`，`mean_class_accuracy` 适用于所有动作识别数据集，`mmit_mean_average_precision` 适用于 Multi-Moments in Time 数据集，`mean_average_precision` 适用于 Multi-Moments in Time 和单类 HVU 数据集，`AR@AN` 适用于 ActivityNet 数据集等。
```
Comment (innerlee) [inline docs_zh-CN/getting_started.md:66]: Moments in Time?

File: docs_zh-CN/getting_started.md [hunk 5]
```diff
@@ -1,3 +1,249 @@
...
+- [3D 人手关键点检测](/docs/tasks/3d_hand_keypoint.md)
+- [2D 人脸关键点检测](/docs/tasks/2d_face_keypoint.md)
+- [2D 全身人体关键点检测](/docs/tasks/2d_wholebody_keypoint.md)
+- [2D 服饰关键点检测](/docs/tasks/2d_fashion_landmark.md)
+- [2D 动物关键点检测](/docs/tasks/2d_animal_keypoint.md)
+
+## 使用预训练模型进行推理
+
+MMPose 提供了一些测试脚本用于测试数据集上的指标（如 COCO, MPII 等），
+并提供了一些高级 API，以便更好地兼容其他 OpenMMLab 项目。
```
Comment (innerlee) [inline docs_zh-CN/getting_started.md:38]: reads strange

File: docs_zh-CN/getting_started.md [hunk 6]
```diff
@@ -1,3 +1,249 @@
...
+    [--gpu-collect] [--tmpdir ${TMPDIR}] [--options ${OPTIONS}] [--average-clips ${AVG_TYPE}] \
+    [--launcher ${JOB_LAUNCHER}] [--local_rank ${LOCAL_RANK}]
+```
+
+此处的 `CHECKPOINT_FILE` 可以是本地的模型权重文件的路径，也可以是模型的下载链接。
+
+可选参数:
+
+- `RESULT_FILE`：输出结果文件名。如果没有被指定，则不会保存测试结果。
+- `--fuse-conv-bn`: 是否融合BN和Conv层。该操作会略微提升模型推理速度。
```
Comment (innerlee) [inline docs_zh-CN/getting_started.md:65]: space before and after `bn`, `conv`

File: docs_zh-CN/getting_started.md (hunk 3 above)
Comment (innerlee) [reply docs_zh-CN/getting_started.md:67]: It is not corrected yet

File: docs_zh-CN/getting_started.md [hunk 7]
```diff
@@ -1,3 +1,249 @@
...
+- `--seed ${SEED}`：设置 python，numpy 和 pytorch 里的种子 ID，已用于生成随机数。
+- `--deterministic`：如果被指定，程序将设置 CUDNN 后端的确定化选项。
+- `--cfg-options CFG_OPTIONS`:覆盖配置文件中的一些实验设置。比如，可以设置'--cfg-options model.backbone.depth=18 model.backbone.with_cp=True'，在线修改配置文件内容。
+- `--launcher ${JOB_LAUNCHER}`：分布式任务初始化启动器选项。可选值有 `none`，`pytorch`，`slurm`，`mpi`。特别地，如果被设置为 `none`, 则会以非分布式模式进行测试。
+- `--autoscale-lr`:根据 [Linear Scaling Rule](https://arxiv.org/abs/1706.02677)，当 GPU 数量或每个 GPU 上的视频批大小改变时，用户可根据批大小按比例地调整学习率。
+- `LOCAL_RANK`：本地 rank 的 ID。如果没有被指定，则会被设置为 0。
+
+`resume-from` 和 `load-from` 的区别：
+`resume-from` 加载模型参数和优化器状态，并且保留检查点所在的周期数，常被用于恢复意外被中断的训练。
+`load-from` 只加载模型参数，但周期数从 0 开始计数，常被用于微调模型。
```
Comment (innerlee) [inline docs_zh-CN/getting_started.md:180]: 周期？

Comment (innerlee) [reply docs_zh-CN/getting_started.md:180]: or 轮数？

File: docs_zh-CN/getting_started.md [hunk 8]
```diff
@@ -1,3 +1,249 @@
...
+如果用户的多台机器通过 Ethernet 连接，则可以参考 pytorch [launch utility](https://pytorch.org/docs/stable/distributed.html#launch-utility)。如果用户没有高速网络，如 InfiniBand，速度将会非常慢。
+
+### 使用单台机器启动多个任务
+
+如果用使用单台机器启动多个任务，如在有 8 块 GPU 的单台机器上启动 2 个需要 4 块 GPU 的训练任务，则需要为每个任务指定不同端口，以避免通信冲突。
+
+如果用户使用 `dist_train.sh` 脚本启动训练任务，则可以通过以下命令指定端口
+
+```shell
+CUDA_VISIBLE_DEVICES=0,1,2,3 PORT=29500 ./tools/dist_train.sh ${CONFIG_FILE} 4
```
Comment (innerlee) [inline docs_zh-CN/getting_started.md:214]: We should make  ports in dist_train.sh random

Comment (jin-s13) [reply docs_zh-CN/getting_started.md:214]: Maybe in another PR.

File: docs_zh-CN/getting_started.md (hunk 7 above)
Comment (jin-s13) [reply docs_zh-CN/getting_started.md:180]: 训练轮数

File: docs_zh-CN/getting_started.md (hunk 8 above)
Comment (innerlee) [reply docs_zh-CN/getting_started.md:214]: yeah

**Review by innerlee — APPROVED** [review]

**Pull request merged**